### PR TITLE
[4.1] Remove the math.div deprecation

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

Kinda replaces #36906 (no FA upgrade to v6)

### Summary of Changes
- patch the font awesome files
- local fixes

### Testing Instructions
- Apply this Pr's branch
- run npm ci
- Check that nothing is broken visually and there are no console deprecation when running any node (css related) cli command

### CODE REVIEW

File: build/build-modules-js/init/patches.es6.js [hunk 1]
```diff
@@ -29,4 +29,21 @@ module.exports.patchPackages = async (options) => {
   let ShortandsweetJs = await readFile(shortandsweetPath, { encoding: 'utf8' });
   ShortandsweetJs = ShortandsweetJs.concat('shortAndSweet(\'textarea.charcount\', {counterClassName: \'small text-muted\'});');
   await writeFile(shortandsweetPath, ShortandsweetJs, { encoding: 'utf8', mode: 0o644 });
+
+  // Patch the Font Awesome math.div sass deprecations
+  // _larger.scss
+  let faPath = join(mediaVendorPath, 'fontawesome-free/scss/_larger.scss');
+  let newScss = (await readFile(faPath, { encoding: 'utf8' })).replace('(4em / 3)', '(4em * .333)').replace('(3em / 4)', '(3em * .025)');
```
Comment (brianteeman) [inline build/build-modules-js/init/patches.es6.js]:
```suggestion
  let newScss = (await readFile(faPath, { encoding: 'utf8' })).replace('(4em / 3)', '(4em * .333)').replace('(3em / 4)', '(3em * .25)');
```

corrects the math

**Comment by brianteeman** [comment]
I can confirm that the deprecation notices have now gone

~Is it intentional that the joomla.asset.json files no longer have a version hash?~

Commented on the code where there is an error in your math

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 5f01580d852bcbf3eb0a7fbfbc52535b3d25fe08<br /><br />Thank you<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37255">issues.joomla.org/tracker/joomla-cms/37255</a>.</sub>

**Comment by dgrammatiko** [comment]
> Is it intentional that the joomla.asset.json files no longer have a version hash?

The hash is not generated automatically per `npm ci`, you need to run specifically `npm run versioning`. (the reason is because for development having to invalidate this hash every time you apply some changes in any of the css/js would be an extra step which will be a way slower workflow)

File: build/build-modules-js/init/patches.es6.js (hunk 1 above)
Comment (dgrammatiko) [reply build/build-modules-js/init/patches.es6.js]: Thanks

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 5f01580d852bcbf3eb0a7fbfbc52535b3d25fe08<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37255">issues.joomla.org/tracker/joomla-cms/37255</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37255">issues.joomla.org/tracker/joomla-cms/37255</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**